tests/test_pipreqs.py

```diff
@@ -37,50 +47,51 @@ def setUp(self):
             "bs4",
             "after_method_is_valid_even_if_not_pep8",
         ]
-        self.modules2 = ["beautifulsoup4"]
-        self.local = ["docopt", "requests", "nose", "pyflakes"]
-        self.project = os.path.join(os.path.dirname(__file__), "_data")
-        self.empty_filepath = os.path.join(self.project, "empty.txt")
-        self.imports_filepath = os.path.join(self.project, "imports.txt")
-        self.imports_no_version_filepath = os.path.join(self.project, "imports_no_version.txt")
-        self.imports_any_version_filepath = os.path.join(self.project, "imports_any_version.txt")
-        self.non_existent_filepath = os.path.join(self.project, "non_existent_file.txt")
-
-        self.parsed_packages = [
+        cls.modules2 = ["beautifulsoup4"]
+        cls.local = ["docopt", "requests", "nose", "pyflakes", "ipython"]
```

Comment:
why is ipython being added here, not related to this commit. (and maybe not necessary at all?)

Reply:
(cross-contamination)

Reply:
Fixed.